### PR TITLE
Remove ETL

### LINKED ISSUE #2959: Remove ETL and document alternative
The ETL project is no longer something we want to spend time thinking about and are planning to drop it (move it to another repo for where it will hopefully fade into obsolescence). ETL being one of the most common and most important operations when working with large amounts of data and rasters almost always being large amounts of data, we'll need to provide alternative mechanisms and updated documentation that guides users towards writing code that is a bit less sui generis.

These tasks need to happen for us to tear out the ETL module:

- [x] Review documentation for ETL and update it with direction to ETL's new home (gt-contrib)
- [x] Surface pipeline documentation as an alternative (it is currently not advertised in the docs very well)
- [x] Document how RasterSources can make life easier and provide [example code](https://github.com/pomadchin/vlm-performance/blob/662f17d8de8d053074c1c7c358ed8e9433138d00/src/main/scala/geotrellis/contrib/performance/IngestRasterSource.scala)
- [x] ETL archived elsewhere

### PR DESCRIPTION
**Pull request by moradology**
## Overview

This PR has 3 objectives:
1. :fire: the ETL project;
2. Banish it to https://github.com/geotrellis/etl
3. Document an alternative path to ingests

### Checklist

- [x] `docs/CHANGELOG.rst` updated, if necessary
- [x] `docs` guides update, if necessary

Closes #2959

### CODE REVIEW

**Comment by moradology** [comment]
@jbouffard I've gone ahead and rebased on top of the spark split; we can switch the target branch to master once you've rebased off it and merged things in

File: docs/guide/pipeline.rst [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-Pipelines are the operative construct in GeoTrellis (the original idea was taken from [PDAL](https://pdal.io/pipeline.html)).
-Pipelines can represent not a simple ETL process but a set of instructions: how to read data, transform (process),
-write it. It is possible to do just with a common GeoTrellis API, but we decided
-to create a separate package to simplify some common processing tasks
-and to reduce code amount that should be written to perform some common and simple operations.
+Pipelines are an idea originally inspired by [PDAL](https://pdal.io/pipeline.html) pipelines.
+Pipelines represent a set of instructions: how to read data, transform (process) said data, and
+write it. It is possible to do this with other parts of the GeoTrellis API, but the pipeline provides
+an alternative which could simplify some common processing tasks and to reduce the amount of code
+that necessary to perform some common operations.
```
Comment (jbouffard) [inline docs/guide/pipeline.rst]: That  *is* necessary

File: docs/guide/ingests.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
+Ingesting Imagery with GeoTrellis
+=================================
+
+A common problems faced by those with large amounts of data is how it
```
Comment (jbouffard) [inline docs/guide/ingests.rst]: *problem*

**Comment by moradology** [comment]
@pomadchin This isn't what you have in mind? https://github.com/locationtech/geotrellis/pull/2969/files#diff-caf2a6b8f4947d018f68893c695b5202R125

**Pull request merged**